docs/crio.conf.5.md

```diff
@@ -476,6 +476,9 @@ CRI-O reads its configured registries defaults from the system wide containers-r
 **separate_pull_cgroup**=""
   [EXPERIMENTAL] If its value is set, then images are pulled into the specified cgroup.  If its value is set to "pod", then the pod's cgroup is used.  It is currently supported only with the systemd cgroup manager.
 
+**automatically_reload_registries**=false
+ Enable/Disable the automatic reloading of mirror registry in CRI-O when there is an update to the 'registries.conf.d' directory. Default value is set to 'false'.
```

Comment:
This sentence is different than the one for the command-line switch, should both be identical?